docs/kadena-docs-style-guide.md

```diff
@@ -0,0 +1,579 @@
+---
```

Comment:
should we put this file in the root of the docs package?
with a link in the readme?

afraid that this document will get lost.